entity/c/README.md

```diff
@@ -0,0 +1,89 @@
+# Prerequisites
+
+-   OpenSSL (TODO: Elaborate why OpenSSL is needed).
```

Comment:
How about briefly explaining why OpenSSL is needed?

Reply:
**THIS IS RELATED TO THE COMMENT ABOVE OF SENDER_BUF SIZE

Currently the send message does not support dynamic sizes, the max length is shorter than 1024 because this 1024 includes header, encryptions, hmacs etc.

When dynamic buffer sizes are needed, most of the code is implemented in malloc and free, but not when making the sender buf. Do you recommend to change the total connection part to dynamic memory allocation or only when sending messages?

In more details, for example, the auth_hello_reply message's size is dynamic due to the sender_buf(e.g. net1.client) and purpose_buf(e.g. {"group":"Servers"}, {"keyId": 00000000}). 
On the other hand, the buffer size when sending handshake1 from client to server is static(Actually changes by encryption scheme, but not implemented yet.). 

So, what do you recommend to use when sending bufs? Static of dynamic? If static how should I decide the size of the buffer.